### ISSUE TITLE
Ubuntu 14 - error while loading shared libraries

### ISSUE THREAD

**Issue by tomatau** [cypress-io/cypress-cli#2]
Attempting to run cypress for CI on jenkins:
```
Description:    Ubuntu 14.04.3 LTS
Linux 3.13.0-36-generic x86_64
```
```
$ cypress -V
0.6.0

$ cypress install
  - Downloading Cypress [==============================] 100% 0.0s
  - Unzipping Cypress   [==============================] 100% 0.0s
  - Finished Installing /var/lib/jenkins/.cypress/Cypress

$ cypress verify
{ [Error: Command failed: /bin/sh -c /var/lib/jenkins/.cypress/Cypress/Cypress --smoke-test --ping=0.6650748795364052
/var/lib/jenkins/.cypress/Cypress/Cypress: error while loading shared libraries: libnotify.so.4: cannot open shared object file: No such file or directory
]
  killed: false,
  code: 127,
  signal: null,
  cmd: '/bin/sh -c /var/lib/jenkins/.cypress/Cypress/Cypress --smoke-test --ping=0.6650748795364052' }
```

**Comment by thomas-holmes** [comment]
You probably just need to install the appropriate libnotify package. I think on ubuntu it is `libnotify-bin` or `libnotify-dev`

If not, just check apt for libnotify packages, one of them should have the appropriate lib.

**Comment by tomatau** [comment]
That definitely helped make progress

Now getting:
```
{ [Error: Command failed: /bin/sh -c /var/lib/jenkins/.cypress/Cypress/Cypress --smoke-test --ping=0.2901066541671753
[27163:0918/093950:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[27163:0918/093950:ERROR:browser_main_loop.cc(216)] Gtk: cannot open display:
[27165:0100/000000:ERROR:zygote_linux.cc(585)] write: Broken pipe
]
  killed: false,
  code: 1,
  signal: null,
  cmd: '/bin/sh -c /var/lib/jenkins/.cypress/Cypress/Cypress --smoke-test --ping=0.2901066541671753' }
```

**Comment by brian-mann** [comment]
This is happening because Cypress requires `xvfb` be installed to run headlessly.

All you need to do is install `xvfb`, and under the hood Cypress will start / run / stop `xvfb` as it needs. As long as `xvfb` is available in your system, everything should work.

https://csshyamsundar.wordpress.com/2011/07/07/installing-xvfb-on-ubuntu/

**Comment by brian-mann** [comment]
I can additionally upgrade the CLI tool to check for this - it actually does, but I can see it's not hitting the `catch`. 

It would be nice to automatically point you to an article that explains how to install `xvfb`, or maybe just provide the commands for you in the error output.

**Comment by tomatau** [comment]
xvfb is already installed, I thought it would be being called from node-xvfb without the need for xvfb-run, we're already using xvfb-run with nightwatch.js tests for a different app.

**Comment by brian-mann** [comment]
You shouldn't have to start the xvfb process yourself - the CLI tool should do that for you automatically. There's probably a bug somewhere.

**Comment by tomatau** [comment]
FWIW this is what we're running in CI

```
gulp build --config=test
gulp serve:dist &
cypress ci <id>
kill %1
```

**Comment by brian-mann** [comment]
Ok, should be fixed by: 60f05d83f57d0ccaf4be345aecc7b612d598ea8f

Go ahead and see if this works in your CI setup.

The problem was `cypress ci` was doing an initial verification of the app, and wasn't wrapping that by starting / stopping `xvfb`.

I've simplified the initial verification to only check whether the Cypress exists as the expected path, and either running or installing it based on the result.
